lessons/Early-Math/identity-property/langs/en-us/pages/scale-rotate-identity.adoc

```diff
@@ -0,0 +1,99 @@
+= Scale, Rotate and the Identity Property
+
+++++
+<style>
+#content td {padding: 0rem 0px !important}
+#content table .autonum::after { content: ')'; }
+#content th { text-align: center !important; }
+</style>
+++++
+
+Use @starter-file{scale-rotate} to respond to the questions below. Refer to the contacts as needed.
+
+
+[.table1, cols="1,1", grid="none", frame="none", stripes="none"]
+|===
+
+| @show{(contract "scale" '(("factor" "Number") ("img" "Image"))
+    "Image")}
+| @show{(contract "rotate" '(("degree" "Number") ("img" "Image"))
+    "Image")}
+|===
+
+== Scale
+
+@n Click “Run”, then type `hello` into the interactions area of @proglang. Hit Enter. What does @proglang return? @fitb{}{}
+
+@fitb{}{}
+
+@n Type @show{(code '(scale 2 "hello"))}. What happened? How is the image returned different from what you saw for Q1? @fitb{}{}
+
+@fitb{}{}
+
+@n The Identity Property tells us that multiplying or dividing an expression by 1 does not change its value. Sometimes, `scale` produces an identical result, too: scaling by 1 will always produce an image that is identical to the original image! Try scaling by 1 in @proglang to confirm.
+
+@n With your partner, predict what belongs in each blank below. Then, test your prediction using @proglang to see if you were correct. We’ve completed the first one for you.
```

Comment:
Oh. Is this confusing? I wanted to fill in ONE of the blanks for each statement so that it's not entirely open-ended. I thought leaving the underlines would make it clearer, but perhaps not.

Reply:
Yeah, definitely confusing. Just remove the underline where the answer is filled in, and tell students to simply "fill in the blanks"